dmscripts/generate_questions_csv.py

```diff
@@ -95,9 +118,16 @@ def generate_csv(output_directory, framework_slug, content_loader, context):
     for index, row in enumerate(rows):
         row.extend([''] * (max_length - len(row)))
 
-    with open('{}/{}-questions.csv'.format(output_directory, framework_slug), 'wb') as csvfile:
-        writer = csv.writer(csvfile, delimiter=b',', quotechar=b'"')
-        header = ["Page title", "Page title hint", "Question", "Hint"]
+    filename = framework_slug
```

Comment:
The problem with generating the name, was that it didn't take into account the context you passed in on the command line. So I realised the task was then to generate a filename from a (potentially, if not in practice) arbitrarily complex YAML expression, which was a bit eww. It also makes for less surprises in terms of trashing a file you didn't expect. I'll remove the obsolete code.